infra/build/developer-tools/build/scripts/task_helper_functions.sh

```diff
@@ -375,8 +375,11 @@ function check_tflint() {
       fi
       # load default ruleset
       tflintCfg="/root/tflint/.tflint.example.hcl"
+      # load if local repo ruleset
+      if [[ -f "./github/.tflint.repo.hcl" ]]; then
+        tflintCfg="./github/.tflint.repo.hcl"
       # if module, load tighter ruleset
-      if [[ $path == "." || $path == "./modules"* ]];then
+      elif [[ $path == "." || $path == "./modules"* || $path =~ "^[0-9]-.*" ]]; then
```

Comment:
What is this supposed to match: ""^[0-9]-.*"?

Reply:
Hey @g-awmalik This pattern is used in a few repos like: https://github.com/terraform-google-modules/terraform-example-foundation

e.g. `2-environments`